src/api/SFA.DAS.Provider.Events.Api/Controllers/SubmissionsController.cs

```diff
@@ -59,5 +60,34 @@ public SubmissionsController(IMediator mediator, ILogger logger)
                 return InternalServerError();
             }
         }
+
+        [Route("api/v2/submissions", Name = "GetLatestLearnerEventByStandard")]

```

Comment:
There doesn't appear to be any unit tests against this controller action.
Also of concern is the class 'SubmissionEventEntity' is being exposed directly as a result. There is no API Type and hence no API Client Code to call this endpoint. (This may be a problem with some existing endpoints, but I haven't checked)
I'm not clear on how ValidationException could be thrown. I was unable to test the endpoint locally, so I could not test is this is thrown. This failure to test locally, is partly down to the CloudService Role being mis-configured (pre-existing issue).

Reply:
Looking into changing the type being returned and adding unit tests now.

Reply:
Latest push to this branch adds unit tests for the Controller Action, changes the return type to be the correct one, updates the API Client to include the new Action (and added client tests) and Validation is now handled properly.

Reply:
Looks good to me, just one question. How do you run the solution locally as my fails when trying to run just the API?